crates/bevy_render/src/lib.rs

```diff
@@ -579,13 +579,12 @@ fn apply_extract_commands(render_world: &mut World) {
 /// If the [`RenderAdapter`] is a Qualcomm Adreno, returns its model number.
 ///
 /// This lets us work around hardware bugs.
-pub fn get_adreno_model(adapter: &RenderAdapter) -> Option<u32> {
+pub fn get_adreno_model(adapter_info: &AdapterInfo) -> Option<u32> {
```

Comment:
```suggestion
pub fn get_adreno_model(adapter_info: &RenderAdapterInfo) -> Option<u32> {
```
You should use `RenderAdapterInfo` from Bevy rather than directly the wgpu type
and also change the doc, the imports, ...

Reply:
I didn't even realize this was a public method, I though it was just an internal helper function.
Done: https://github.com/bevyengine/bevy/pull/19355/commits/3fbce03c0b08ab963eef477956dbb2dc2fdf66d9

Is it correct that I'm creating the `RenderAdapterInfo(WgpuWrapper::new(info))` where I need it and just have the wgpu adapter?